docs/source/packages.rst

```diff
@@ -20,10 +20,11 @@ The links below lead to the package-specific documentation, including a complete
     :alt: PyPI signac
 
 The *core* **signac** package implements a simple, serverless, distributed database directly on the file system.
-It allows you manage files on the file system and associate them with JSON-encoded metadata.
-This metadata is immediately searchable, which allows you to find and select data for specific data sub spaces.
+Its features include:
 
-This package also provides the data management functionality for **signac-flow** and **signac-dashboard**.
+- Manage project data with a well-defined indexable storage layout for data and metadata.
+- Search and manipulate the data in existing **signac** projects.
+- Provide the data management functionality for **signac-flow** and **signac-dashboard**.
```

Comment:
```suggestion
- Unlock the use of **signac-flow** and **signac-dashboard** for additional utility.
```
Phrase the dependency as what the user gets out of it

Reply:
I'm not sure if unlock is the right word here, it might confuse the users that the core signac package is dependent on signac-flow and signac-dashboard. What do you think about this?

Reply:
I want to show that signac-flow and dashboard depend on signac. But I also want to keep a parallel grammatical structure to what the user can do. Maybe enable instead of unlock?
"Enable the use of signac-flow and signac-dashboard " ?
As in: you have to use signac to enable the other packages.

**Actually, I think we can just get rid of this last bullet -- dependencies can be dealt with on the installation page** Even with these revisions it doesn't tell me much about what the user gets out of the package

Reply:
To show the dependency between these packages, we mention that _"You only have to install those that provide the functionality you need, however both **signac-flow** and **signac-dashboard** require the **signac** core package."_ in the introductory paragraph of `packages` page

Reply:
Good point! I felt safe to remove that bullet then